src/pebble.py

```diff
@@ -257,6 +266,35 @@ def _mjolnir_pebble_layer(self) -> ops.pebble.LayerDict:
         }
         return typing.cast(ops.pebble.LayerDict, layer)
 
+    @property
+    def _irc_bridge_pebble_layer(self) -> ops.pebble.LayerDict:
+        """Generate pebble config for the irc bridge service.
+
+        Returns:
+            The pebble configuration for the irc bridge service.
+        """
+        command_params = (
+            f"-c {synapse.IRC_BRIDGE_CONFIG_PATH}"
+            + f" -f {synapse.IRC_BRIDGE_REGISTRATION_PATH}"
+            + f" -p {synapse.IRC_BRIDGE_HEALTH_PORT}"
```

Comment:
Is the `+` required here? I think it should work without it and it will get auto-merged by the compilation step

Reply:
True, omitted.

Reply:
TIL also, thanks!